gadget/mountedfilesystem.go

```diff
@@ -838,7 +868,7 @@ func (f *mountedFilesystemUpdater) rollbackFile(dstRoot, source, target string,
 	sameStamp := backupPath + ".same"
 	preserveStamp := backupPath + ".preserve"
 
-	if strutil.SortedListContains(preserveInDst, dstPath) && osutil.FileExists(preserveStamp) {
+	if osutil.FileExists(preserveStamp) {
```

Comment:
what's the effect of missing the Contains check?

Reply:
Hmm that's some bad patch splitting on my side. I'll force push the change here.